contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -1212,6 +1212,26 @@ boolean CreateProcessW(String lpApplicationName, char[] lpCommandLine,
      *         GetLastError.
      */
     HANDLE OpenProcess(int fdwAccess, boolean fInherit, int IDProcess);
+    
+    /**
+     * This function retrieves the full path of the executable file of a given process.
+     * 
+     * @param hProcess
+     * 			Handle for the running process
+     * @param dwFlags
+     * 			0 - The name should use the Win32 path format.
+     * 			1(PROCESS_NAME_NATIVE) - The name should use the native system path format. 
+     * @param lpExeName
```

Comment:
Add definition of _int PROCESS_NAME_NATIVE = 0x00000001;_ flag definition to _WinBase_ (or wherever it is usually defined)